dbt/models/default/schema/default.vw_pin_universe.yml

```diff
@@ -81,6 +81,15 @@ models:
         description: '{{ doc("column_chicago_police_district_num") }}'
       - name: class
         description: '{{ doc("shared_column_class") }}'
+        tests:
+          - count_is_consistent:
+              name: default_vw_pin_universe_class_count_is_consistent_by_year
+              group_column: year
+              config:
+                error_if: ">25"
```

Comment:
These tests are not new, I've simply moved them from table-level tests to the columns that they are operating on.